cmd/kots/cli/get-config.go

```diff
@@ -209,18 +210,24 @@ func decryptGroups(clientset kubernetes.Interface, namespace string, groups []v1
 	for _, group := range groups {
 		outGroup := group.DeepCopy()
 		for idx, item := range group.Items {
-			if item.Type == "password" {
+			if item.Type == "password" && decrypt {
 				// attempt to decrypt the password's value and default
-				decrypted, err := decryptString(item.Value.String())
-				if err == nil {
+				if item.Value.String() != "" {
+					decrypted, err := decryptString(item.Value.String())
+					if err != nil {
+						continue // don't fail if we can't decrypt
```

Comment:
Not sure we want continue here. If `value` fails, we still probably want to decode `default`?